### DISCUSSION TITLE
Rowy- fetching data for story book

### DISCUSSION THREAD

**Discussion by ghost** [AntlerVC/firetable#1331] [Support / Q&A] [answered]
I created a table and added a prompt but no data is fetched and table stays empty. What can I do?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @greenauraproduct. Just to clarify, are you not seeing a response object generated by OpenAI when you enter your prompt? One possible reason could be that you haven't configured your secrets with the correct Secret Name. To address this, I recommend following the step-by-step guide provided in this blog post: https://www.rowy.io/blog/storybooks-bravo.

If this still doesn't work, could you share a recording via https://www.jam.dev/ so that we can take a look?
